src/app/features/edit-nonce-drawer/edit-nonce-drawer.tsx

```diff
@@ -33,17 +31,10 @@ export function EditNonceDrawer() {
   const { errors, setFieldError, setFieldValue, validateField, values } = useFormikContext<
     StacksSendFormValues | StacksTransactionFormValues
   >();
-  const { data: nextNonce } = useNextNonce();
   const [loadedNextNonce, setLoadedNextNonce] = useState<number | string>();
   const navigate = useNavigate();
 
-  useOnMount(() => {
-    // Make sure the nonce value is set when the form is launched bc at times
-    // the next nonce is slow to load and is undefined when the initial
-    // form values are set by the send form
-    if (isUndefined(values.nonce)) setFieldValue('nonce', nextNonce?.nonce);
-    setLoadedNextNonce(values.nonce);
-  });
+  useOnMount(() => setLoadedNextNonce(values.nonce));
```

Comment:
I realized we used the `NonceSetter` component to handle this.

Reply:
so why do we need any nonce setting code at all?

Reply:
I feel like I have answered this question before? Sometimes the nonce doesn't load in time to set the initial form value.

Reply:
It's been discussed, indeed. But you've pointed out that this logic is happening elsewhere. So I'm thinking: if we have a component to set the nonce, it's loading and hasn't resolved yet, the user goes to the edit nonce drawer, why do we need to repeat the same logic that's already happening in the `NonceSetter`? If we wait, won't it get set anyway?

Reply:
Sorry, I think I commented on the wrong line here so might have confused you? I am removing code here bc I realized I need to use the `NonceSetter` component and added it around the forms. I am removing L44 here with the comment. L37 is just setting local component state to track what the nonce is when loaded.

Reply:
Notice the `NonceSetter` component was added around the Stacks forms in this PR.